torch/ao/quantization/fx/convert.py

```diff
@@ -316,7 +316,11 @@ def convert_standalone_module(
         produce a reference model or a fbgemm/qnnpack model
       - backend_config_dict: backend configuration of the target backend of quantization
     """
-    convert = torch.ao.quantization.quantize_fx.convert_fx  # type: ignore[attr-defined]
+    # TODO: remove is_reference flag
```

Comment:
not sure if we can do this, probably need discussion about how do we handle is_reference here in standalone modules, for example, one option is to always convert to reference quantized model and do lowering in a separate step

Reply:
Yes, though I think we would need to refactor the lowering part out of `torch.ao.quantization.fx.convert` before we can do that